src/main.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  copyTemplateFiles,
+  createProjectDirectory,
+  installPackages,
+  initGitRepository,
+} from "./tasks";
+import type { Options } from "./types";
+import { renderOutroMessage } from "./utils/render-outro-message";
+import chalk from "chalk";
+import Listr from "listr";
+import path from "path";
+import { fileURLToPath } from "url";
+
+export async function createProject(options: Options) {
+  console.log(`\n`);
+
+  const currentFileUrl = import.meta.url;
+
+  const templateDirectory = path.resolve(
+    decodeURI(fileURLToPath(currentFileUrl)),
+    "../../templates"
+  );
+
+  const targetDirectory = path.resolve(process.cwd(), options.project);
+
+  const tasks = new Listr([
+    {
+      title: `📁 Create project directory ${targetDirectory}`,
+      task: () => createProjectDirectory(options.project),
+    },
+    {
+      title: `🚀 Creating a new Scaffold-ETH 2 app in ${chalk.green.bold(
+        options.project
+      )}`,
+      task: () =>
+        copyTemplateFiles(options, templateDirectory, targetDirectory),
+    },
+    {
```

Comment:
changing order to `initializing git` -> `installing deps` fixes my errors even for `yarn cli`